tasks/task03.py

```diff
@@ -0,0 +1,61 @@
+texts = [
+   "Hello, World!",
+   "The world is mine",
+   "Hello, how are you?"
+]
+
+def count_words():
+  all_words = []
+
+  for x in texts:
+    words = x.lower().split(" ")
+    all_words.extend(words)
+  return all_words
+
+
+def collect_words_frequencies(words):
+  dictionary = {}
+
+  if words is not None:
+    for item in words:
+      word = remove_symbol(item)
+
+      if word in dictionary:
+        dictionary[word] = dictionary[word] + 1
+      else:
+        dictionary[word] = 1
+    return dictionary
+  else:
+    return None
+
+
+def remove_symbol(item):
+  word = str(item)
+  word = word.replace(",", "")
+  word = word.replace("?", "")
+  word = word.replace(".", "")
+  word = word.replace("!", "")
+  return word
+
+
+def find_first_occurrence(word):
+  for i in range(len(texts)):
+    words = texts[i].split(" ")
+    for w in words:
+      compare = remove_symbol(w).lower()
```

Comment:
`import re`

`compare = re.sub(r"[^a-zA-Z0-9]", " ", word)`

and delete `remove_symbol`

Reply:
thanks